File: Língua do P.py
frase = input()
tamanho = len(frase)
i = 0
while True:
    if i >= tamanho:
        break
    if frase[i] == 'p':
        print(frase[i+1], end='')
        i += 2
    else:
        print(frase[i], end='')
        i += 1

File: Língua do P.cpp
#include <iostream>

using namespace std;

int main(){
	string frase;

	getline(cin, frase);

	int tamanho = frase.size(), marcador = 0;

	while(true){
		if(marcador >= tamanho){
			break;
		}
		else if(frase[marcador] == 'p'){
			cout << frase[marcador+1];
			marcador += 2;
		}
		else{
			cout << frase[marcador];
			marcador++;
		}
	}

	cout << endl;

	return 0;
}

